### ISSUE TITLE
Need to account for umask in builds

### ISSUE THREAD

**Issue by dje4g** [effenel/build#5]
I did a fresh build and noticed /bin,/usr, and a few others in the root image had mode 750.
I'm guessing this is because my umask at the time of the build was 027.

**Comment by kulakowski** [comment]
This also ends up breaking builds, because init refuses to load init.rc et al if the permissions are too loose.

**Comment by dje4g** [comment]
I was wondering if a simple algorithm would handle all but a handful of cases, and the remaining ones we'd want to explicitly handle anyway (so as to not have to trust the makefile or whatever, and it'd all be in one place). Then we wouldn't have to worry about umask during builds, and could fix it in finalize_rootimg.sh or some such.
[we'd still be making *some* assumption about umask, but only to the extent that the build completes]

**Comment by kulakowski** [comment]
Yeah, I think the issue should really be "be explicit about permissions in the image" rather than requiring a certain umask per se.
